docs/USING_ADVANCED.md

```diff
@@ -13,6 +13,31 @@ const marked = new Marked([options, extension, ...]);
 |:--------|:-------|:----------------------------------------------------------------------|
 | options |`object`|The same arguments that can be passed to [`marked.use`](/using_pro#use)|
 
+Another case is, if you want to rerun function everytime they open a file, or you want to make marked.run() non-state, you can create a new instance of Marked to ensure options and extensions are locally scoped.
```

Comment:
It doesn't seem like this is saying anything more than the documentation above. That the user should use Marked instance if they want extensions to be locally scoped.

I think the example should emphasize that `marked.use(...)` should not be used in a loop or function. It should only be used directly after `new Marked` is created or `marked` is imported